Mathport/Syntax/Translate/Tactic/Mathlib/Misc1.lean

```diff
@@ -243,7 +243,9 @@ open AST3 Mathport.Translate.Parser
   let cfg ← liftM $ (← expr?).mapM trExpr
   match c with
   | none => `(tactic| tauto $[(config := $cfg)]?)
-  | some _ => `(tactic| tauto! $[(config := $cfg)]?)
+  | some _ =>
+    warn! "tauto! no longer supported, replacing with tauto"
```

Comment:
I think we don't need the warning here. The new `tauto` is equivalent to the old `tauto!`, and the old `tauto` is just a half baked weak version of the same thing (it claimed to be classical-avoiding but it wasn't really).

Reply:
Good point.  Changed.